deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -120,7 +126,7 @@ spec:
           failureThreshold: 3
         readinessProbe:
           httpGet:
-            path: /healthz
+            path: /readyz
```

Comment:
Right. this is good. Do we have an issue adding the readiness check for controller ?

Reply:
There is no specific issue - this has apparently been added as a part of https://github.com/Kong/kubernetes-ingress-controller/pull/1314/files

Reply:
Gotcha. 
we need follow k8s readiness probes and define the function reporting the readiness. https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes. The good thing is we already having the path defined in the pod.